math/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "wasm-math"
```

Comment:
I think that this name is too wide for this functionality, what about `u256-math` or even `u256`? And also suggest to rename the project root directory in that way.